source/people/louishartono.rst

```diff
@@ -0,0 +1,24 @@
+Louis Evantio Hartono
+=====================
+
+
+Contact Information
+-------------------
+
+| 1423 Monroe St Apt #302
```

Comment:
I don’t think you need to give your home address here.   Either use your office number or don’t use any address.